README.md

```diff
@@ -1,2 +1,2 @@
-# the-real-cost
-Opportunity Cost Application
+# The Real Cost Application
+Marshall and Jesper's IOS project
```

Comment:
I would prefer my name to have my last name.

Reply:
I'm not doing that. First names only.